profiler/src/ProfilerEngine/Datadog.Profiler.Native/DllMain.cpp

```diff
@@ -44,6 +44,11 @@ extern "C" BOOL STDMETHODCALLTYPE DllMain(HINSTANCE hInstDll, DWORD reason, PVOI
 
 bool CheckProfilingEnabledEnvironmentVariable()
 {
+#ifdef ARM64
+    Log::Warn("Profiler is deactivated because it runs on an unsupported architecture.");
```

Comment:
NIT: I think we should evaluate the environment variable like we do in the other platforms, and only log the warning if the result is "profiler should be enabled". This way we won't add non-actionable warnings to customers who didn't enable the profiler anyway

Reply:
Maybe should have also updated the official datadog documentation. There is no mention anywhere that arm is not supported.